ultra/ultra/utils/episode.py

```diff
@@ -321,6 +321,7 @@ def episodes(n, etag=None, log_dir=None):
             f"Total Steps",
             f"Steps/Sec",
             f"Score",
+            f"Goal completion",
```

Comment:
```suggestion
            f"Goal Completed",
```
So that it matches the table title format of: https://github.com/huawei-noah/SMARTS/blob/4e4957b26b6d56e527c4831eba4c7eb0defa16bb/ultra/ultra/utils/episode.py#L321 Additionally, maybe changing it to "Goal Completed" (instead of "Goal Completion") would imply the True/False result we intend (see other review comment).